### PR TITLE
Learn - Value Transfer Update

### PR DESCRIPTION
**Pull request by charlesthompson3**
# Description of change

Content in the Learn: Value Transfer section has been updated in lieu of the Stardust updates.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix
- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project

### CODE REVIEW

File: internal/learn/about-iota/value-transfer.md [hunk 1]
```diff
@@ -6,82 +6,82 @@ description: Learn how the IOTA Cryptocurrency Protocol transfers the native IOT
...
+### The Goal
 
 Charlie wants to transfer 10i to Bob.
 
 ### What is in the Tangle
 
 In this example, Alice had 10i that she transferred to Bob. Therefore, the Tangle now includes a value message that transferred those 10i to Bob.
 
-### Choosing where to attach messages
+### Choosing Where to Attach Messages
```
Comment (lzpap) [inline internal/learn/about-iota/value-transfer.md:23]: I don't get the `In this example, Alice had 10i that she transferred to Bob` part. Where does Alice ocme into play when the goal is that Charlie tranfers 10i to Bob?

Comment (lzpap) [reply internal/learn/about-iota/value-transfer.md:23]: So what is confusing is that:
 - We state that Alice transferred 10i to Bob.
 - The example is about how Charlie transfers 10i to Bob.
 - Why the hell do we need Alice at all then?

Let's get rid of this Alice transferred bob thingy. And later, when you make the double spend example just say: but what if Charlie submitted two trasanctions, one that transfers Bob 10i and one that transfers Alice 10i?

**Review by jlvandenhout — APPROVED** [review]

**Comment by Dr-Electron** [comment]
Why is this a stardust doc?

**Comment by charlesthompson3** [comment]
> Why is this a stardust doc?

Small node operation update/correction, I believe.

**Comment by jlvandenhout** [comment]
Both @Dr-Electron and I think we can just target the `main` branch here, because the changes are more editorial than Stardust related changes, if that's okay with you @charlesthompson3?

**Comment by charlesthompson3** [comment]
> Both @Dr-Electron and I think we can just target the `main` branch here, because the changes are more editorial than Stardust related changes, if that's okay with you @charlesthompson3?

Branch switched and ready for approval.

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**